earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -3,6 +3,11 @@
 TMPFILE="/tmp/FusionTutorial-Full.tar.gz"
 SCRIPTDIR=`dirname "$0"`
 
+if [ $(id -u) != 0 ] ; then
+    echo "`basename $0`" "must be run as root"
```

Comment:
Sorry, I should've given an example. `basename $0` won't work for a script whose name contains any special character.  You can, e.g.:

```BASH
echo "$(basename "$0") must be run as root"
```

or, I usually define `SELF_NAME` near the top of the file, and the use that variable:

```BASH
SELF_NAME=$(basename "$0")

# . . .

echo "$SELF_NAME must be run as root"
```